### ISSUE TITLE
Still working?

### ISSUE THREAD

**Issue by enoversum** [noahbuscher/N1-Jiffy#9]
Hi there,

had a very old version of Jiffy installed, now installed the latest. While the button to insert a gif was there on the old version, nothing happened by clicking the icon anymore. Removed the old version, downloaded and installed the new version instead, but now not even the button appears. Is this plugin still being developed atm? Running the latest N1 on Windows 10 x64.

Thanks in advance :wink: !

**Comment by noahbuscher** [comment]
Interesting! I'll look into this! Thank you.

**Comment by rahulghanate** [comment]
+1 for the same issue.
Latest NylasN1-0.4.33 and Jiffy 0.1.6, shows icon in compose dialog but the pop-up stopped to display.
So upgraded Jiffy to 0.1.8, but now the "Gif" icon is also missing.

**Comment by rahulghanate** [comment]
The issue seems to be due to installation steps.
I earlier installed it by copying the extracted N1-Jiffy directory in .nylas/packages/ directory
I uninstalled the Jiffy plugin from the 'Preferences'->'Plugins' menu.
Then I installed it back with 'Developers'->'Install a plugin' option, and directed it to the extracted directory.
Now the compose window showing the Gif icons and also the popup window. 

It's an awesome plugin.
